### PR TITLE
Completed string-manipulation-1 exercise - Cathy Garcia.

### PR DESCRIPTION
**Pull request by cathyygrcia**
Screen Shot 2023-07-14 at 12 22 29 PM
Screen Shot 2023-07-14 at 12 22 46 PM
Screen Shot 2023-07-14 at 12 22 52 PM

### CODE REVIEW

File: string-manipulation-1/src/is-lower-cased.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] !== word[i].toLowerCase()) {
+      return false;
+    }
+  }
+  return true;
+}
```
Comment (shawnkost) [inline string-manipulation-1/src/is-lower-cased.js:9]: We can actually accomplish this without a loop!
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```

File: string-manipulation-1/src/is-upper-cased.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] !== word[i].toUpperCase()) {
+      return false;
+    }
+  }
+  return true;
+}
```
Comment (shawnkost) [inline string-manipulation-1/src/is-upper-cased.js:9]: Same note as above ^

File: string-manipulation-1/src/is-vowel.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    case 'o':
+      return true;
+    case 'U':
+      return true;
+    case 'u':
+      return true;
+    default:
+      return false;
+  }
+}
```
Comment (shawnkost) [inline string-manipulation-1/src/is-vowel.js:27]: If we convert `char` to lowercase, we can simplify this. That way we only need to check for lowercase values.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```

**Review by shawnkost — APPROVED** [review]
Approved, but check out my comments!

File: string-manipulation-1/src/is-vowel.js (hunk 3 above)
Comment (cathyygrcia) [reply string-manipulation-1/src/is-vowel.js:27]: if we use .toLowerCase(), how does it check for the uppercase? Also, do I not need a return statement after each case??

**Pull request merged**